classes/jats/exceptions/UnableToCreateJATSContentException.php

```diff
@@ -14,12 +14,17 @@
 namespace PKP\jats\exceptions;
 
 use Exception;
+use Throwable;
 
 class UnableToCreateJATSContentException extends Exception
 {
 
-    public function __construct(public ?Exception $innerException = null)
+    public function __construct(public ?Throwable $innerException = null)
     {
-        parent::__construct(__('publication.jats.defaultContentCreationError'), null, $innerException);
+        parent::__construct(
```

Comment:
Should these changes be in a different PR?

Reply:
> Should these changes be in a different PR?

that is true but because of this I was having some issue with some of workflow page loading to run some tests

Reply:
It's already been resolved in `main` in 8ded2f38ebb1572bf0cf47568c51c18684de4a58.

Reply:
Then it will be removed at next rebase

Reply:
This is still showing up in the PR.

Reply:
I kept it because we are passing second param as `null` at [here](https://github.com/pkp/pkp-lib/blob/main/classes/jats/exceptions/UnableToCreateJATSContentException.php#L23) which is not allowed as that should be a `int` as specified [here](https://www.php.net/manual/en/class.exception.php)